cypress/locators/app-wrapper/locators.js

```diff
@@ -1,2 +1,3 @@
 // component preview locators
-export const APP_WRAPPER_PREVIEW = '.carbon-app-wrapper';
+const APP_WRAPPER_PREVIEW = ".carbon-app-wrapper";
+export default APP_WRAPPER_PREVIEW;
```

Comment:
shouldn't there be an empty line?

Reply:
There is, it just doesn't show on Github